src/views/ConfigurationUIView.vue

```diff
@@ -152,4 +156,8 @@ const interfaceStore = useAppInterfaceStore()
 const updateOpacity = (value: number): void => {
   interfaceStore.setBgOpacity(value)
 }
+
+const resetColorsToDefault = (): void => {
+  interfaceStore.setUIGlassEffect(0.8, '#4F4F4F1A', '#FFFFFF', 25)
```

Comment:
It would be good to interesting to concentrate those default values in the `defaults.ts` file, so it gets easy to find and anyone can import from there if needed.